dev/tools/dartdoc.dart

```diff
@@ -89,13 +89,22 @@ dependencies:
 /// paths. Cleanup unused index.html files no longer needed.
 void createIndexAndCleanup() {
   print('\nCreating a custom index.html in $kDocRoot/index.html');
+  removeOldFlutterDocsDir();
   renameApiDir();
   copyIndexToRootOfDocs();
   addHtmlBaseToIndex();
   putRedirectInOldIndexLocation();
   print('\nDocs ready to go!');
 }
 
+void removeOldFlutterDocsDir() {
+  try {
+    new Directory('$kDocRoot/flutter').deleteSync(recursive: true);
+  } catch(e) {
```

Comment:
It would have been better to use the `on` keyword to capture only the specific exception that is thrown when the directory exists, but I wouldn't worry too much about it.